tests/utils.py

```diff
@@ -549,3 +550,38 @@ def mocked_aws_s3_bucket_test_file(bucket_name, file_name, file_content="Test fi
         tmp_file.flush()
         s3.upload_file(Bucket=bucket_name, Filename=tmp_file.name, Key=file_name)
     return "s3://{}/{}".format(bucket_name, file_name)
+
+
+def mocked_http_file(test_func):
+    # type: (Callable[[...], Any]) -> Callable
+    """
+    Mocked http file for http reference test
```

Comment:
Please provide a better description. It should indicate what is the purpose/outcome of using this mock.

Refer to `mocked_aws_s3` description. 
There should be a note regarding the specific value defined by `MOCK_HTTP_REF`.

Reply:
Tried to explain better, can you take a look at the updated comment.

Reply:
New one is good.